src/services/email/handlers/processEmails.tsx

```diff
@@ -0,0 +1,116 @@
+import { DateTime } from "luxon";
+import { decode } from "base-64";
+import {
+  CognitoIdentityProviderClient,
+  ListUsersCommand,
+  // UserType as CognitoUserType,
+} from "@aws-sdk/client-cognito-identity-provider";
+import { SESClient, SendTemplatedEmailCommand } from "@aws-sdk/client-ses";
+
+import { KafkaEvent } from "shared-types";
+
+// import { CognitoUserAttributes } from "shared-types";
+
+const SES = new SESClient({ region: process.env.region });
+const Cognito = new CognitoIdentityProviderClient({
+  region: process.env.region,
+});
+
+const emailsToSend = {
+  "initial-submission-medicaid-spa": [{
+    "templateBase": "initial-submission-medicaid-spa-cms",
+    "sendTo": [process.env.osgEmail],
+  }, {
+    "templateBase": "initial-submission-medicaid-spa-state",
+    "sendTo": ["submitterEmail"],
+  }],
+  "initial-submission-chip-spa": [{
+    "templateBase": "initial-submission-chip-spa-cms",
+    "sendTo": [process.env.chipEmail],
+    "addCC": [`${process.env.chipCCList}`],
+  }, {
+    "templateBase": "initial-submission-chip-spa-state",
+    "sendTo": ["submitterEmail"],
+  }],
+}
+
+const createSendTemplatedEmailCommand = (data) =>
+  new SendTemplatedEmailCommand({
+    Source: process.env.emailSource ?? "kgrue@fearless.tech",
+    Destination: {
+      ToAddresses: data.ToAddresses,
+      CcAddresses: data.CcAddresses,
+    },
+    TemplateData: JSON.stringify({ applicationEndpoint: "onemac.cms.gov", ...data }),
```

Comment:
Note to self: applicationEndpoint replacement value should come from environment variable (ui-infra?) to be specific to the branch.

Reply:
Done and tested!